specification.md

```diff
@@ -3713,31 +3696,23 @@ Let's take a look an an example "onError" definition inside a state:
 <td valign="top">
 
 ```yaml
-onError:
-- expression: "{{ $.errors[?(@.name == 'FunctionExecutionError')] }}"
+onErrors:
+- error: Item not in inventory
```

Comment:
I'm imagining that this is an error that might come out of an open-api based function.  It's not clear to me how the response from a RESTful API will get translated to an error message string here.  Is there even a standard format for error messages from open-api?

btw; this is more for my own understanding, not really a comment on this PR.

Reply:
it does not have to do with openapi. runtime impls have to allow for a mapping between explicit/domain-specific error names + code prop defined in workflows to actual errors.